### DISCUSSION TITLE
Octopus Intelligent - Charge/Discharge cycling during the cheap overnight rate

### DISCUSSION THREAD

**Discussion by gmarchment** [springfall2008/batpred#757] [Q&A]
I can understand why this behaviour exists, as the cheap rate undercuts the export rate, and so predbat will charge, then discharge to maximise revenue, and ensure 100% charge ahead of 05:30 when the cheap rate ends.

<img width="576" alt="image" src="https://github.com/springfall2008/batpred/assets/151647330/89731dfd-2498-4750-953f-ad93ccef3c80">

However, I wondered if this kind of power cycling is detrimental to the battery, or against any terms of use with Octopus?  

Is there a setting to prevent this?  Or should I just enjoy the extra cash I'll receive as a result?

**Comment by springfall2008** [comment]
You could either

A) change metric battery cycle which models the cost of using your battery. Be careful as higher settings will result in larger bills and less battery use.

B) turn off calculate discharge on charge which will prevent most of this 

C) pocket the cash

**Reply by gmarchment** [reply]
Thanks Trefor :) Appreciate the guidance on B especially.  Will probably leave it as C though!

**Reply by iainfogg** [reply]
@gmarchment I use option B as otherwise it seems to be against the spirit of the Intelligent tariff even if it's not necessarily against the rules.

But I do a full dump and recharge each day, on the basis that the battery has a full warranty for unlimited discharges.

Each to their own though 😃

**Comment by RobinCu** [comment]
As for whether it’s detrimental, lots of opinions and nothing official from GE.  I think the warranty is 10-12 years depending on battery or 10,000 full cycles 0%-100%-0%.  My view is to use it, as otherwise it’s like paying £7,000 to re-carpet the house and then sealing up a room to avoid wearing the new carpet out.
